spec/features/01_user_sign_in_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature "sees sign in options" do
+  scenario "see all sign in options" do
+    visit user_index_path
+    expect(page).to have_content('Sign In')
+    expect(page).to have_content('Sign Out')
+  end
+
+  scenario "user successfully signs in" do
+    user = FactoryGirl.create(:email)
+    user_password = FactoryGirl.create(:password)
```

Comment:
Aren't these attributes on the `User` model?? You should probably just create a `User` object, and just use their email and password.